es6-shim.js

```diff
@@ -265,8 +267,12 @@
 
   var overrideNative = function overrideNative(object, property, replacement) {
     var original = object[property];
+    if (typeof original == 'undefined'){
```

Comment:
Please avoid `==`, `===` is needed here. Also, please put a space between `)` and `{`.